acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java

```diff
@@ -125,13 +125,12 @@ public static boolean isMinus(String minus) {
             fis = new FileInputStream(new File(path));
             return ymal.loadAs(fis, clazz);
         } catch (Exception e) {
-            throw new AclException(String.format("The  file for Plain mode was not found , paths %s", path), e);
```

Comment:
Only ignore the FileNotFoundException

Reply:
Okay,I will adjust my codes in next commit.

Reply:
Great！